2.7/Dockerfile

```diff
@@ -7,7 +7,7 @@ MAINTAINER SoftwareCollections.org <sclorg@redhat.com>
 EXPOSE 8080
 
 ENV PYTHON_VERSION=2.7 \
-    PATH=$HOME/.local/bin/:$PATH \
+    PATH=$HOME/.local/bin/:/opt/app-root/node_modules/.bin:$PATH \
```

Comment:
@torsava no reason other than taking it from  [redhat-developer/s2i-dotnetcore](https://github.com/redhat-developer/s2i-dotnetcore/commit/b8992565ca6f2724103cfdfbc049a1806929bfe4), but I will dig deeper into this if I get green light to actually add npm into this image